### PR TITLE
Add Perdido examples

### PR DESCRIPTION
**Pull request by wldcordeiro**
This displays usage of Perdido in JSS.

### CODE REVIEW

File: perdido/app.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
+(function() {
```
Comment (kof) [inline perdido/app.js:1]: I intentionally separate the setup and styles to encourage users to do the same, see https://github.com/jsstyles/examples/blob/gh-pages/calendar/components/jss.js

File: perdido/app.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  jss.use(jssVendorPrefixer())
+  jss.use(jssPropsSort())
+
+  var blockMar = 30;
+
+  jss.createStyleSheet({
+    // Reset code
+    '*, *:after, *:before': {
+      boxSizing: 'border-box',
+      padding: '0',
```
Comment (kof) [inline perdido/app.js]: can we relay on jss-default-unit in this example, to make it a bit cleaner ...?

File: perdido/index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            <h1><a href="https://gitub.com/wldcordeiro/perdido">Perdido Grid</a></h1>
+        </header>
+
+        <div class="description">
+            <a href="https://github.com/jsstyles/jss-examples/tree/gh-pages/perdido">View Source</a>
+            <p>Description here.</p>
+        </div>
+
+        <!-- Simple 1/4 -->
+        <section>
```
Comment (kof) [inline perdido/index.html]: I assume you are not using generated classes for simplicity reason. However I fear to make  a bad statement with global selectors.

**Comment by wldcordeiro** [comment]
Bump, @kof this is updated with some changes I think you'll approve of.

File: perdido/section-styles.js [hunk 4]
```diff
@@ -0,0 +1,192 @@
+var blockMar = 30;
+
+window.setupStyles = {
+  // Reset code
+  '*, *:after, *:before': {
+    boxSizing: 'border-box',
+    padding: '0',
```
Comment (kof) [inline perdido/section-styles.js]: this can be number

Comment (wldcordeiro) [inline perdido/section-styles.js]: It could but defaultUnit didn't get the update for unitless zeros out to npm yet so I wrapped it in quotes.

Comment (kof) [inline perdido/section-styles.js]: I will publish it, forgot about it. But besides of that 0px when generated on the client is not a problem at all.

Comment (kof) [inline perdido/section-styles.js]: released

File: perdido/section-styles.js [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+      padding: 30
+    }
+  },
+
+  // Waffle Grid
+  mmmWaffles: {
+    background: '#47E3FF',
+
+    '& article': {
+    background: '#26DEFF',
```
Comment (kof) [inline perdido/section-styles.js]: indentation seems to be broken

**Comment by kof** [comment]
merged

**Pull request merged**